### PR TITLE
Add Regression Test for WPIMULT in ACTIONX Block

### PR DESCRIPTION
**Pull request by bska**
To keep this feature working as intended.

### CODE REVIEW

**Comment by bska** [comment]
jenkins build this please

**Comment by bska** [comment]
Note: The only [failing regression test](https://ci.opm-project.org/job/opm-simulators-PR-builder/3836/testReport/) is the new one introduced in this PR, due to missing reference solutions.  We must create new such solutions through the `update_data` mechanism.

File: tests/update_reference_data.sh [hunk 1]
```diff
@@ -191,6 +191,7 @@ tests[max_wgr_1]="flow wtest/wecon_wgr_max MAX_WGR_1"
 tests[rxft_smry]="flow rxft_smry TEST_RXFT"
 tests[bo_diffusion]="flow diffusion BO_DIFFUSE_CASE1"
 tests[fpr_nonhc]="flow water-1ph WATER2F"
+tests[actionx_wpimult]="flow water-1ph ACTIONX_WPIMULT"
```
Comment (akva2) [inline tests/update_reference_data.sh:194]: this looks wrong.

Comment (bska) [reply tests/update_reference_data.sh:194]: > this looks wrong.

Good catch.  I'll fix it immediately.

**Comment by akva2** [comment]
jenkins build this update_data please

**Comment by akva2** [comment]
jenkins build this opm-tests=788 please

**Pull request merged**